tensorwaves/physics/helicityformalism/kinematics.py

```diff
@@ -274,15 +284,15 @@ def register_subsystem(self, subsystem: SubSystem):
             They can be used to retrieve the kinematic data from the dataset
             returned by :meth:`~convert`.
         """
-        state_fs = []
+        state_fs: list = []
         for fs_uid in subsystem.final_states:
             state_fs += fs_uid
         invmass_name = self.register_invariant_mass(list(set(state_fs)))
         angle_names = self.register_helicity_angles(subsystem)
 
         return (invmass_name,) + angle_names
 
-    def _convert_ids_to_indices(self, ids: tuple):
+    def _convert_ids_to_indices(self, ids: tuple) -> Sequence:
```

Comment:
Here, I guess the output should be `Tuple` again, as it's the same as the input.

Generally, I have the impression that `Sequence` is used if you for instance don't know the whether the input is `List` or `Tuple`, but you don't use `Sequence` for output if you have control over the output.

Reply:
Yeah, I'm aware of that. That function is return either one or the other though. But let me see if I can change that.

Reply:
Ok changed to tuple now